ansible/actions/catalog-db-solr-sync.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: 'catalog-harvester'
+  gather_facts: false
+  tasks:
+
+    - name: Get package name
+      pause:
+        prompt: >
+          What dataset would you like to sync between the DB and SOLR?
+           You can set this by running this playbook with `--extra-vars
+           "db_solr_sync_package_name=insert-package-name-here"`
+      when: db_solr_sync_package_name is undefined
+      register: pkg_name
+
+    - name: Get input
+      set_fact:
+        db_solr_sync_package_name: "{{ pkg_name.user_input }}"
+      when: db_solr_sync_package_name is undefined
+
+    - name: Run Sync Command
+      shell: "ckan search-index rebuild {{ db_solr_sync_package_name }}"
```

Comment:
`command` is preferred to `shell`, unless you actually need shell syntax (`||`, `&&`, `|`, etc)